src/tetris/field/remoteField.ts

```diff
@@ -75,5 +100,41 @@ export default class RemoteField {
 			}
 		}
 	}
+
+	private _setupText() {
+		this._name = this._scene.add.text(this._drawOffset.x, this._drawOffset.y, " ", config.ui.fonts.scoreboard.font);
+		this._score = this._scene.add.text(this._drawOffset.x, this._drawOffset.y, " ", config.ui.fonts.scoreboard.font);
+		this._adjustTextX();
+		this._adjustTextY();
+	}
+
+	private _updateSprites(serializedBlockState) {
```

Comment:
Data type missing (`Array<Array<SerializedBlockState>>`).
Also the parameter name could be more precise (e.g. serializedFieldState)